build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -180,6 +187,12 @@ stages:
         artifactName: Linux Packages
         targetPath: $(System.DefaultWorkingDirectory)/bin/Build$(XA.Build.Configuration)/linux-artifacts
 
+    - task: PublishPipelineArtifact@1
+      displayName: publish nupkgs
+      inputs:
+        artifactName: nupkgs
```

Comment:
I think we'll need a unique name here to distinguish between mac and linux produced nupkg artifacts, as the OSS macOS build is using the same name:
https://github.com/xamarin/xamarin-android/blob/c94e1ab5c87a188948a77bbd1eca70e6b672fa3c/build-tools/automation/azure-pipelines-oss.yaml#L111

Reply:
I put the word `Linux` in front of it now.